apps/src/dance/blocks.js

```diff
@@ -92,7 +93,7 @@ const customInputTypes = {
       const newField = new CdoFieldDanceAi();
       currentInputRow
         .appendField(inputConfig.label)
-        .appendField(newField, 'VALUE');
+        .appendField(newField, inputConfig.name);
```

Comment:
Made this change to be consistent with other fields and ensure we're using the field name as defined in the block config.